docs/isoSPI.md

```diff
@@ -1,3 +1,109 @@
-# AMDC isoSPI Interface
+# AMDC isoSPI and Differential I/O Interface
 
-This document describes the design considerations and implementation details for the isoSPI interface subsystem on the Advanced Motor Drive Controller (AMDC).
+This document describes the design considerations and implementation details for the isoSPI and differential I/O interface on the Advanced Motor Drive Controller (AMDC). The block diagram of the implementation is presented and each block is discussed in detail. Details on the PCB layout, and connector pinout are also provided in this document.
+
+
+## Relevant Versions of AMDC Hardware
+
+AMDC REV D
+
+
+## Design Requirements and Considerations
+
+The isolated serial peripheral interface (isoSPI) has increased noise immunity and safety. The following describes the design requirements for the isoSPI and differential I/O (D_IO) interface. The design requirements for the isoSPI and differential I/O (D_IO) interface are as follows:
+
+1. Implement four isoSPI and four differential I/O interfaces.
+
+2. Implement a voltage level translator to interface with the FPGA that operates at 1.8V.

```

Comment:
I don't agree with the presentation of these requirements. Please think about this from the AMDC user's perspective. What does the user want to learn about in this section? 

Take a look at the design requirements listed for the [power stack](https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/PowerStack.md#design-requirements-and-considerations) and [encoder](https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/Encoder.md#design-requirements)

Reply:
I have updated this section.